Api/ride.py

```diff
@@ -1,39 +1,35 @@
 import json
```

Comment:
You should have a docstring comment at the beginning of every module with a brief description of what the module does.